lib/rng.js

```diff
@@ -0,0 +1,8 @@
+export const happensWithAChanceOf = percentageChance => {
+    const drawnNumber = Math.floor((Math.random() * 100) + 1);
+    return drawnNumber <= percentageChance
+        ? true
+        : false
+};
+export const botRefuses = () => happensWithAChanceOf(1);
+export const chooseRandom = list => list[Math.floor(Math.random() * list.length)];
```

Comment:
what is this for?

Reply:
It's for functions like Dear Viktor where it returns random answer from the list.